src/plugins/saved_objects/public/saved_object_finder.tsx

```diff
@@ -44,19 +42,13 @@ import {
 import { Direction } from '@elastic/eui/src/services/sort/sort_direction';
 import { i18n } from '@kbn/i18n';
 
-import { SavedObjectAttributes } from '../../../../core/public';
-import { SimpleSavedObject, CoreStart } from '../../../../core/public';
-import { useKibana } from '../context';
-
-// TODO the typings for EuiListGroup are incorrect - maxWidth is missing. This can be removed when the types are adjusted
-const FixedEuiListGroup = (EuiListGroup as any) as React.FunctionComponent<
```

Comment:
`EuiListGroup` already contains `maxWidth`, so, I think, we can remove the wrapper `FixedEuiListGroup`.